pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.constants.tsx

```diff
@@ -0,0 +1,8 @@
+import { Operators } from './DBClusterBasicOptions.types';
+
+export const OPERATORS = [Operators.xtradb, Operators.psmdb];
+
+export const DatabaseOperators = {
+  [Operators.xtradb]: 'MySQL',
```

Comment:
DATABASE_LABELS in shared/core can be used here.

Reply:
done